### ISSUE TITLE
File Attributes Overwritten and Desktop Items Invisible after 'dconf update'.

### ISSUE THREAD

**Issue by danila-Skachedubov** [mate-desktop/caja#1769]
#### Expected behaviour
A method on dbus is provided that implements the functionality of pressing the F5 button for the desktop.

#### Actual behaviour
1. Executing the dconf update command results in the overwriting of file attributes.
2. New files or folders created on the desktop are not visible until the F5 key is pressed.
3. Pressing the F5 key after executing dconf update reveals previously created folders and files.

#### Steps to reproduce the behaviour
1.  Execute the dconf update command.
2. Create a new file or folder on the desktop.
3. Observe that the created items are not visible.
4. Press the F5 key.
5. After pressing F5, observe that previously created files and folders appear.

#### MATE general version
1.28.0

#### Package version
1.28.0

#### Linux Distribution
all

**Comment by danila-Skachedubov** [comment]
Should we expect to see this behavior corrected?

**Comment by lukefromdc** [comment]
I don't think that is something we can fix from inside MATE, but rather is a dconf issue but I do not know enough about dconf to be sure. Leaving this open at the moment so any other team members can weigh in

**Comment by danila-Skachedubov** [comment]
Perhaps we could mention someone from the development team who could assist with this?

**Comment by lukefromdc** [comment]
I do not know which team members that may be, and remember this is NOT a MATE bug
